### PR TITLE
Add 'logs', 'describe' buttons to tree view, remove 'get'.

### PR DESCRIPTION
**Pull request by djwhatle**
**For resources that have user-facing logs, I added a new `oc logs` command**

The copied command is built such that it will filter specifically for logs for the selected resource. This takes advantage of structured logging keys from mig-controller.
image

---

**For all resources, I swapped out the `oc get` command for a more informative `oc describe` command that shows the resource contents and associated events in a single view.**
image

Todo:

- [x] Hide the log copy button on resources that don't have directly connected logs (routes, jobs, PVCs, PVs, PVBs, PVRs)

### CODE REVIEW

File: src/app/home/pages/PlanDebugPage/components/TreeActionsDropdown.tsx [hunk 1]
```diff
@@ -41,46 +45,94 @@ const TreeActionsDropdown: React.FunctionComponent<ITreeActionsDropdownProps> =
...
+        command
+      </span>
+    </DropdownItem>
+  );
+
+  const logsDropdownButton = (
+    <DropdownItem
+      key="copy-oc-logs-command"
+      onClick={(event: React.MouseEvent<HTMLAnchorElement>) => {
+        onCopyLogs(event);
```
Comment (ibolton336) [inline src/app/home/pages/PlanDebugPage/components/TreeActionsDropdown.tsx:93]: setKebabIsOpen(false);

File: src/app/home/pages/PlanDebugPage/components/TreeActionsDropdown.tsx [hunk 2]
```diff
@@ -41,46 +45,94 @@ const TreeActionsDropdown: React.FunctionComponent<ITreeActionsDropdownProps> =
...
+    );
+  };
+
   const encodedPath = encodeURI(rawNode.objectLink);
 
+  const describeDropdownButton = (
+    <DropdownItem
+      key="copy-oc-describe-command"
+      onClick={(event: React.MouseEvent<HTMLAnchorElement>) => {
+        onCopyDescribe(event);
```
Comment (ibolton336) [inline src/app/home/pages/PlanDebugPage/components/TreeActionsDropdown.tsx:76]: setKebabIsOpen(false);

**Review by ibolton336 — CHANGES_REQUESTED** [review]
Seeing the dropdown stay open after making a selection. Otherwise looks good.

**Comment by djwhatle** [comment]
@ibolton336 fixed the kebab, also changed the popup message to accurately reflect what is being copied (`oc describe` or `oc logs`, not `oc get` anymore).

One other thing: with the tree providing the cluster, we don't need to hard-code the cluster type returned from the helper function `getOCCommandAndClusterType`, we can pull it off the `rawNode`. I'll plan to fix that in a follow-on PR.

**Comment by djwhatle** [comment]
Rebased

File: src/app/home/pages/PlanDebugPage/helpers.ts [hunk 3]
```diff
@@ -38,97 +38,137 @@ export const getFullKindName = (kind: any) => {
...
-        clusterType: '',
+        ocDescribeCommand: `oc describe ${getFullKindName(kind)} -n ${namespace} ${name}`,
+        ocLogsCommand: null,
       };
     case 'Pod':
       return {
-        ocCommand: `oc get ${getFullKindName(kind)} -n ${namespace} ${name}`,
-        clusterType: '',
+        ocDescribeCommand: `oc describe ${getFullKindName(kind)} -n ${namespace} ${name}`,
+        ocLogsCommand: `oc logs -n ${namespace} ${name}`,
```
Comment (ibolton336) [inline src/app/home/pages/PlanDebugPage/helpers.ts:126]: Does this one need a container specified also?

Was seeing this output when trying to run the oc logs command 
``` oc logs -n mediawiki dvm-rsync-vdmpn
error: a container name must be specified for pod dvm-rsync-vdmpn, choose one of: [rsync-client stunnel]
```

**Review by ibolton336 — CHANGES_REQUESTED** [review]

File: src/app/home/pages/PlanDebugPage/helpers.ts (hunk 3 above)
Comment (djwhatle) [reply src/app/home/pages/PlanDebugPage/helpers.ts]: Resolved in latest commit 👍 , I also made `-n` longform as `--namespace` in all generated commands so the user can have a better chance of understanding what's going on.

**Comment by djwhatle** [comment]
Rebased, thanks @ibolton336 I think this is ready to merge now.

**Review by ibolton336 — APPROVED** [review]
LGTM!

**Pull request merged**